scripts/create-component/package-template/package.json

```diff
@@ -22,6 +22,7 @@
   "dependencies": {{dependencies}},
   "peerDependencies": {
     "@pluralsight/ps-design-system-normalize": "*",
+    "@pluralsight/ps-design-system-theme": "*",
```

Comment:
Do these peerDeps w/ versions of "*" get published that way, or does it publish with a concrete (eg, "3.0.4") version?

Reply:
doesn't look like lerna does the versioning for us 😢 

i'll go through and update